example/hello-world/README.md

```diff
@@ -0,0 +1,21 @@
+# Hello World Command Spin App
+
+## Prerequisites
+
+1. Install [`cargo-component`](https://github.com/bytecodealliance/cargo-component) with a version that is compatible with WASI 0.2.0
+(at least `v0.7.0`).
+
+2. Install the command plugin from the releases or build building and installing from main:
+
+```sh
+cd ..
```

Comment:
This would need to be either `cd ../..` or potentially we could put these plugin prereq and build steps in the base README.md and then just keep the `Running` section here?  Either way!

Reply:
Good call! Updated